core-bundle/contao/templates/forms/form_captcha.html5

```diff
@@ -18,9 +18,7 @@
   <input type="hidden" name="<?= $this->name ?>_hash" value="<?= $this->hasErrors() ? $this->getHash() : '' ?>">
 
   <?php if (!$this->hasErrors()): ?>
-    <?php $this->addCspHash('style-src', 'display:none'); ?>
-    <?php $this->addCspSource('style-src', 'unsafe-hashes'); ?>
-    <div style="display:none">
+    <div style="<?= $this->cspInlineStyle('display:none') ?>">
```

Comment:
What do you think about renaming the method to just `inlineStyle()`, so our users do not have to know about CSP?

Reply:
Well, in the last call we discussed that users _must_ know about CSP. But since the other functions are called `nonce`, `addHash` and `addSource` I guess for consistency this could be `inlineStyle`, yes.